doc/mgr/osd_support.rst

```diff
@@ -0,0 +1,60 @@
+OSD Support Module
+==================
+The OSD Support module holds osd specific functionality that
+is needed by different components like the orchestrators.
+
+In current scope:
+
+* osd draining
```

Comment:
Maybe I'm bad a googling but I fail to find any description in the ceph docs as to what is `drain` is really about.
I guess that it is an attempt to tell the OSD to move its data to other OSDs without stalling PGs, but I guess it would not hurt to put that explicitly in the doc.

Reply:
`draining` probably isn't a well established term in the ceph universe.. yet :)

> I guess that it is an attempt to tell the OSD to move its data to other OSDs without stalling PGs, but I guess it would not hurt to put that explicitly in the doc.

Correct, it gracefully moves data from away from a specific OSD.

I'll add a sentence or two about `draining`

Reply:
done